scripts/python/junglecatsoftware/awstoolkit/outputFormatters.py

```diff
@@ -20,7 +20,7 @@ def printSmallHeader( function = 'Output Formatter' ):
   print('\n=={}=='.format(function))
 
 def printStartDateStamp():
-  print( 'Begin at {0:%Y-%m-d %H:%M:%S.%f}'.format( datetime.datetime.now() ) )
+  print( 'Begin at {0:%Y-%m-%d %H:%M:%S.%f}'.format( datetime.datetime.now() ) )
```

Comment:
The worst part... I actually had caught that myself, but managed not to fix it somehow (I thought I did). D'oh! But that's what tests are for!